ghcide/src/Development/IDE/Types/Exports.hs

```diff
@@ -30,15 +30,20 @@ import           HieDb
 import           Name
 import           TcRnTypes                   (TcGblEnv (..))
 
-newtype ExportsMap = ExportsMap
-    {getExportsMap :: HashMap IdentifierText (HashSet IdentInfo)}
-    deriving newtype (Monoid, NFData, Show)
+data ExportsMap = ExportsMap
+    {getExportsMap :: HashMap IdentifierText (HashSet IdentInfo)
+    , getModuleExportsMap :: Map.HashMap IdentifierText [Text]
```

Comment:
```suggestion
    , getModuleExportsMap :: Map.HashMap ModuleNameText [Text]
```

Reply:
I think the keys are module names right? Better change the type synonym to reflect that and even better, add Haddock comments.

Reply:
Can we have a `HashSet IdentInfo` as the value type, for consistency with `getExportsMap`?

Reply:
resolved